release/models/rib/openconfig-rib-bgp-tables.yang

```diff
@@ -311,6 +322,826 @@ submodule openconfig-rib-bgp-tables {
+        reference "RFC9136: IP Prefix Advertisement in Ethernet VPN (EVPN)";
+
+        uses bgp-evpn-type-five-key-refs;
+
+        container state {
+          description
+            "Operational state data for IP Prefix Advertisement L2VPN EVPN
+            route entries in the BGP LOC-RIB";
+
+          leaf ethernet-tag {
+            type oc-evpn-types:ethernet-tag;
+            description
+              "The Ethernet tag identifying the broadcast domain for this
+              route";
+          }
+
+          leaf ip-prefix-length {
+            type string;
+            description
+              "The ip-prefix length for the route";
+          }
+
+          leaf ip-prefix {
+            type oc-inet:ip-prefix;
+            description
+              "The ip-prefix for the route";
+          }
+
+          uses bgp-loc-rib-attr-state;
+          uses bgp-loc-rib-route-annotations-state;
+         }
+
+         uses bgp-evpn-route-path-type5-state;
+      }
+    }
+  }
+
+  grouping l2vpn-evpn-adj-rib-top {
+    description
+      "Top-level grouping for L2VPN-EVPN Adj-RIB table";
+
+    container neighbors {
+      config false;
+      description
+        "Enclosing container for neighbor list";
+
+      list neighbor {
+        key "neighbor-address";
+        description
+          "List of neighbors (peers) of the local BGP speaker";
+
+        leaf neighbor-address {
+          type leafref {
+            path "../state/neighbor-address";
+          }
+          description
+            "Reference to the list key";
+        }
+
+        container state {
+          description
+            "Operational state for each neighbor BGP Adj-RIB";
+
+          leaf neighbor-address {
+            type oc-inet:ip-address;
+            description
+              "IP address of the BGP neighbor or peer";
+          }
+        }
+
+        container adj-rib-in-pre {
+          description
+            "Per-neighbor table containing the NLRI updates
+            received from the neighbor before any local input
+            policy rules or filters have been applied.  This can
+            be considered the 'raw' updates from the neighbor.";
+        }
+
+        container adj-rib-in-post {
+          description
+            "Per-neighbor table containing the paths received from
+            the neighbor that are eligible for best-path selection
+            after local input policy rules have been applied.";
+        }
+
+        container adj-rib-out-pre {
+          description
+            "Per-neighbor table containing paths eligble for
+            sending (advertising) to the neighbor before output
+            policy rules have been applied";
+        }
+
+        container adj-rib-out-post {
+          description
+            "Per-neighbor table containing paths eligble for
+            sending (advertising) to the neighbor after output
+            policy rules have been applied";
+        }
+      }
+    }
+  }
+
```

Comment:
Just a general comment - can you please double check that all your closed braces are in the right alignment / indentation? Some of them look off, but it could just be the red / green on the screen that is making it appear to be off (notably the closing } ... ie in 1128 and closing on 1133, it looks like 1133 is one space too soon? Please verify all your closing braces.

Reply:
@chula-benny I looked at the alignment/indentation in my vscode editor and it looks right to me.  I attached a screen shot below.

![Screen Shot 2022-01-26 at 1 52 42 PM](https://user-images.githubusercontent.com/12445795/151227849-f7dd7b66-678b-456f-aa1d-6e26e8f0e780.png)

Reply:
I have found a couple of other areas however where the alignment is off a bit.  I am fixing them now.

Reply:
@chula-benny Fixed the alignment issues and pushed the changes.